metals/src/main/scala/scala/meta/internal/builds/NewProjectProvider.scala

```diff
@@ -0,0 +1,358 @@
+package scala.meta.internal.builds
+
+import scala.collection.mutable
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future
+import scala.util.Try
+
+import scala.meta.internal.metals.ClientCommands
+import scala.meta.internal.metals.ClientConfiguration
+import scala.meta.internal.metals.Icons
+import scala.meta.internal.metals.JsonParser._
+import scala.meta.internal.metals.Messages._
+import scala.meta.internal.metals.MetalsEnrichments._
+import scala.meta.internal.metals.MetalsInputBoxParams
+import scala.meta.internal.metals.MetalsLanguageClient
+import scala.meta.internal.metals.MetalsOpenWindowParams
+import scala.meta.internal.metals.MetalsQuickPickItem
+import scala.meta.internal.metals.MetalsQuickPickParams
+import scala.meta.internal.metals.StatusBar
+import scala.meta.internal.metals.Time
+import scala.meta.io.AbsolutePath
+
+import coursierapi._
+import org.eclipse.lsp4j.ExecuteCommandParams
+
+class NewProjectProvider(
+    buildTools: BuildTools,
+    client: MetalsLanguageClient,
+    statusBar: StatusBar,
+    config: ClientConfiguration,
+    time: Time,
+    shell: ShellRunner,
+    icons: Icons,
+    workspace: AbsolutePath
+)(implicit context: ExecutionContext) {
+
+  private val templatesUrl =
+    "https://github.com/foundweekends/giter8/wiki/giter8-templates.md"
+  private val gitterDependency = Dependency
```

Comment:
it is 'giter' not to be confused with 'gitter.im' ;)

Reply:
ach, changed!